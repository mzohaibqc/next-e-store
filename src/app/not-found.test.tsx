import { render } from '@testing-library/react';
import { expect, test } from 'vitest';
import NotFound from './not-found';

test('NotFound should render without error', async () => {
  const { container } = render(<NotFound />);
  // save snapshot
  expect(container).toMatchSnapshot();
});
