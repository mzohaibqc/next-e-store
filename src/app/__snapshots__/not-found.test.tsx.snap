// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound should render without error 1`] = `
<div>
  <div
    class="flex-1 w-full flex justify-center items-center"
  >
    <div>
      <img
        alt="404"
        data-nimg="1"
        decoding="async"
        height="400"
        loading="lazy"
        src="/_next/image?url=%2Fnot-found.png&w=828&q=75"
        srcset="/_next/image?url=%2Fnot-found.png&w=640&q=75 1x, /_next/image?url=%2Fnot-found.png&w=828&q=75 2x"
        style="color: transparent;"
        width="400"
      />
      <p
        class="text-2xl"
      >
        Sorry! This page does not exists.
      </p>
      <a
        class="text-2xl text-blue-800 hover:underline"
        href="/"
      >
        Go back home
      </a>
    </div>
  </div>
</div>
`;
